docs/C_STYLE_GUIDE.md

```diff
@@ -122,6 +122,29 @@ void tearDown(void)
 
```

Comment:
In a future we should add a section about helper functions here.  Let me know if you want me to do it.
This PR LGTM.

Reply:
I can add it into this PR shortly 👍